cli/ceph/balancer.py

```diff
@@ -0,0 +1,18 @@
+from cli import Cli
+
+
+class Balancer(Cli):
+    """This module provides CLI interface to manage the balancer module."""
+
+    def __init__(self, nodes, base_cmd):
+        super(Balancer, self).__init__(nodes)
+
+        self.base_cmd = f"{base_cmd} balancer"
+
+    def status(self):
+        """Turn On balancer module."""
```

Comment:
To confirm, the status method is to return the status of the balancer? Or is it to turn on the module? If not, please update the method method info here.

Reply:
Thanks @pranavprakash20 
Updated